src/lib/logger.ts

```
@@ -32,6 +33,15 @@ class BreezLogger {
       default:
         console.log(`${prefix}:`, entry.line);
     }
+
+    // Persist Breez log
+    void appendLog({
+      id: Date.now(), // good enough; you can improve later
```

Using `Date.now()` as a unique ID is problematic:
- Multiple logs within the same millisecond will have duplicate IDs
- Since keyPath: `id` is used, `store.put()` will overwrite entries with the same ID, silently losing logs
- High-frequency logging (example: during wallet sync) could easily produce multiple logs per millisecond

Suggestion: Use `autoIncrement: true` in the object store instead:
```
const store = db.createObjectStore(STORE_NAME, {
    keyPath: 'id',
    autoIncrement: true,
});
```
Then remove `id` from the `PersistedLogEntry` type (or make it optional), and let IndexedDB generate unique sequential IDs.

---

I've simplified the log storage by removing the structured log entry type entirely and now store logs as plain formatted strings, as logged to the console, and let IndexedDB handles unique keys automatically as per your suggestion.